src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,39 @@
+import { useParams, Link } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { getArticleById } from '../utils/api';
+
+export const SingleArticle = () => {
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleById(article_id).then((articleData) => {
+      setArticle(articleData);
+    });
+  }, []);
+
+  return (
+    <>
+      <div className='single-article'>
+        <Link to={`/`}>
+          <h3 className='article-header'>{article.title}</h3>
+        </Link>
+        <p>{article.body}</p>
+        <p className='author-text'>
+          <small>
+            By <br />
+            {article.author}
+          </small>
+        </p>
+        <Link to={`/`}>
+          <h6 className='article-header back-btn'>Go Back</h6>
+        </Link>
```

Comment:
I love the idea of having a Go Back button but I'd maybe make it a bit more descriptive. Go Back suggests going to the previous page but if you go to the articles by topic and then view a specific article, I'd assume a Go Back button to take me back to the articles by topic but it takes you to all articles (This is me being very nitpicky though!!)

Reply:
Changed to Home Page so it is more intuitive i think